py/src/ai/h2o/sparkling/ml/algos/SupportsCrossValidation.py

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from ai.h2o.sparkling.ml.params.H2OTypeConverters import H2OTypeConverters
+from pyspark.ml.param import *
+
+
+class SupportsCrossValidation(Params):
+    generateCrossValidationModels = Param(
+        Params._dummy(),
+        "generateCrossValidationModels",
```

Comment:
whether CV models will be generated/created is controlled by nfold parameter, whether they will be saved/exported from H2O - is controlled by this parameters

this makes me thing we need to change the naming, it could be "keepCrossValidationModels" - with similar semantics as "keep_crossvalidation_models" in h2o

Reply:
essentially it does the same thing - it preserves the models

Reply:
no problem, fill fix that. Thank you!